starlite/config/logging.py

```diff
@@ -183,17 +183,11 @@ def configure(self) -> "GetLogger":
         try:
```

Comment:
This `try`/`except` block should probably only go around `from picologging import config, getLogger`